projects/cdk/abstract/dialog.service.ts

```diff
@@ -22,10 +22,13 @@ export abstract class AbstractTuiDialogService<T, K = void> extends Observable<
     }
 
     open<G = void>(
+        // eslint-disable-next-line @typescript-eslint/no-invalid-void-type
```

Comment:
Can you elaborate on that?

Reply:
Bug of typescript-eslint
https://github.com/typescript-eslint/typescript-eslint/issues/6547